packages/utilities/src/aria.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * ARIA helper to concatenate attributes, returning undefined if all attributes
+ * are undefined. (Empty strings are not a valid ARIA attribute value.)
+ *
+ * NOTE: This function will NOT insert whitespace between provided attributes.
+ *
+ * @param ariaAttributes ARIA attributes to merge
+ */
+export function mergeAriaAttributes(...ariaAttributes: (string | undefined)[]): string | undefined {
+  let mergedAttribute = '';
```

Comment:
```return ariaAttributes.filter(arg => arg !== undefined && arg !== null).join('');``` maybe?

Reply:
Sure. I like your idea of `mergeAriaValues` too, I think that would be more clear.